pkg/steps/bundle_source_test.go

```diff
@@ -92,65 +92,64 @@ RUN find . -type f -regex ".*\.\(yaml\|yml\)" -exec sed -i s?quay.io/openshift/o
 RUN find . -type f -regex ".*\.\(yaml\|yml\)" -exec sed -i s?quay.io/openshift/origin-metering-hadoop:4.6?some-reg/target-namespace/stable@metering-hadoop?g {} +
 RUN find . -type f -regex ".*\.\(yaml\|yml\)" -exec sed -i s?quay.io/openshift/origin-ghostunnel:4.6?some-reg/target-namespace/stable@ghostunnel?g {} +`
 
-	fakeClientSet := ciopTestingClient{
-		imagecs: fakeimageclientset.NewSimpleClientset(&apiimagev1.ImageStream{
-			ObjectMeta: v1.ObjectMeta{
+	client := fakectrlruntimeclient.NewFakeClient(
+		&imagev1.ImageStream{
+			ObjectMeta: metav1.ObjectMeta{
 				Namespace: "target-namespace",
 				Name:      api.StableImageStream,
 			},
-			Status: apiimagev1.ImageStreamStatus{
+			Status: imagev1.ImageStreamStatus{
```

Comment:
Might be useful to break out import renames to a separate commit to make review easier